### PR TITLE
Pongo images on action cards fix issue 641

### PR DESCRIPTION
**Pull request by frimpongopoku**
Image problem on card fixed. Switch from percentages to flex box division. 

Prod is back to looking like this 

Screenshot 2021-08-15 at 18 25 15

### CODE REVIEW

File: v2.0/src/Components/Menu/NavBarBurger.js [hunk 1]
```diff
@@ -358,7 +358,10 @@ class NavBarBurger extends React.Component {
   renderLogin() {
     const { links } = this.props;
     const { user } = this.props;
-    const btnColor = user.preferences && user.preferences.color ? user.preferendes.color : "#135dfe";
+    const btnColor =
+      user.preferences && user.preferences.color
+        ? user.preferendes.color
```
Comment (BradHN1) [inline v2.0/src/Components/Menu/NavBarBurger.js:363]: will fix this misspelling

File: v2.0/src/Components/Pages/ActionsPage/PhotoSensitiveAction.js [hunk 2]
```diff
@@ -153,105 +150,97 @@ class PhotoSensitiveAction extends React.Component {
...
   }
 
-  newRender() {
+  newFlexRender() {
     const actionStateCase = this.getActionStateCase();
     return (
       <div
         className={`col-lg-6 col-md-12 col-sm-12 col-12 ${MEAnimation.getAnimationClass()}`}
-        style={{ padding: 10 }}
+        key={this.props.key?.toString()}
```
Comment (BradHN1) [inline v2.0/src/Components/Pages/ActionsPage/PhotoSensitiveAction.js:164]: What does the question mark do in this? 
Just trying to understand javascript better

**Comment by frimpongopoku** [comment]
Hi brad,

It means if. 'key' does not exist? Dont bother running  toString() on it.
It helps prevent errors.

**Review by BradHN1 — APPROVED** [review]
Excellent job!

**Pull request merged**